lib/indexer/addrindexer/addrindexer.js

```diff
@@ -1,7 +1,6 @@
 /*!
  * addrindexer.js - addr indexer
- * Copyright (c) 2014-2015, Fedor Indutny (MIT License)
- * Copyright (c) 2014-2017, Christopher Jeffrey (MIT License).
```

Comment:
These indexers were largely just taken from JJ and Indutny's work right? We should probably keep them in the copyrights if so.